pkg/util/proc/process_unix.go

```diff
@@ -135,6 +135,9 @@ func findProcessSearch(binary, cmdlinePattern string) (*UnixProcess, error) {
 func processes() ([]*UnixProcess, error) {
 	d, err := os.Open("/proc")
 	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, nil
```

Comment:
bah, go-ps doesn't give us the cmdline so updates would be needed to go-ps. Since this is just for unit tests that get mocked anyway, it doesn't seem worth more time than a log statement